docs/getting-started.md

```diff
@@ -88,6 +89,50 @@ We can use the redis image based on alpine linux from the DockerHub.
 The containerd client uses the `Opts` pattern for many of the method calls.
 We use the `containerd.WithPullUnpack` so that we not only fetch and download the content into containerd's content store but also unpack it into a snapshotter for use as a root filesystem.
 
+Let's put the code together and see it in action.
```

Comment:
nit: It would be nice to add a short synopsis of what you expect the following code sample to do.